script/main.py

```diff
@@ -19,10 +19,23 @@ def create_driver(name):
   return driver
 
 
+def list_of_all_links(driver):
+  links_elem = driver.find_elements_by_tag_name("a")
+  return links_elem  
+
+
+def print_list_of_all_links(driver):
+  for value in list_of_all_links(driver):
+    link_value = value.get_attribute('href')    
+    print("name: " + value.text + ", target location: " + link_value)
+
+
 def main():  
   driver, url = fetch_argument()
   driver = create_driver(driver)
   driver.get(url)
+  list_of_all_links(driver)
```

Comment:
What's the point of calling this function here if you don't use its result?
BTW calling is correct, but you should use the result returned by this function.